README.md

```diff
@@ -41,9 +41,10 @@ requests, code review feedback, and also pull requests.
 
 ## Supported Go Versions
 
-We support Go v1.12 and higher.
+Admin Go SDK is compatible with at least the three most recent, major Go releases.
```

Comment:
Truly a nit, but "The Admin Go SDK" reads a little nicer.

Reply:
Good catch. Thanks! Updated.